CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `GammaPrior` can now be chosen as lengthscale prior
 - Serialization user guide
 - Basic deserialization tests using different class type specifiers
+- The variable `user_callable` for `PureRecommender`
```

Comment:
This is a technical detail, you want to focus on what has changed for the user here, so it should be more in the way of "Possibility to provide a user-defined callable for PureRecommender" or something like that.